### ISSUE TITLE
interface referring non-existent interface should give error

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1658]
ref https://github.com/abapGit/abapGit/pull/4394
ref https://github.com/abapGit/ajson_mirror

should respect `errorNamespace`,

```abap
INTERFACE zif_abapgit_ajson PUBLIC.
  INTERFACES zif_ajson_reader.
  INTERFACES zif_ajson_writer.
```
